src/slurm_plugin/clustermgtd.py

```diff
@@ -678,101 +611,54 @@ def _update_static_nodes_in_replacement(self, slurm_nodes):
             if node and not node.is_up():
                 nodes_still_in_replacement.add(nodename)
         self._static_nodes_in_replacement = nodes_still_in_replacement
+        for node in slurm_nodes:
+            node.is_static_nodes_in_replacement = node.name in self._static_nodes_in_replacement
+            node._is_being_replaced = self._is_node_being_replaced(node)
+            node._is_replacement_timeout = self._is_node_replacement_timeout(node)
```

Comment:
Can we move the implementation of `_is_node_being_replaced` and `_is_node_replacement_timeout` into the Node object? IMO there is no reason to keep them in clustermgtd still and initialize attributes of node object like this?

Reply:
After our discussion, we agree to leave the method in clustermgtd since node_replacement_timeout and node_in_replacement should be attributes of clustermgtd